images/installer/Dockerfile.upi.ci.rhel8

```diff
@@ -43,6 +45,9 @@ RUN yum update -y && \
     rm -rf /var/cache/yum/* && \
     chmod g+w /etc/passwd
 
+COPY --from=yq3 /yq /bin/yq-go
+COPY --from=yq4 /usr/bin/yq /bin/yq-v4
+
```

Comment:
We want to remove the curl from the yq below otherwise you're overriding the binary you just copied.

Reply:
ah good catch.  i'll update that.